docs/eventing/samples/ping-source/README.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "PingSource example"
+linkTitle: "PingSource"
+weight: 10
+type: "docs"
+---
+
+This example shows how to configure PingSource as an event source for
+functions.
+
+## Before you begin
+
+1. Set up [Knative Serving](../../../serving).
+1. Set up [Knative Eventing](../../../eventing).
+
+## Create a Knative Service
+
+To verify that `PingSource` is working, create a simple Knative
+Service that dumps incoming messages to its log.
+
+{{< tabs name="create-service" default="By YAML" >}}
+{{% tab name="By YAML" %}}
+Use following command to create the service from STDIN:
+
+```shell
+cat <<EOF | kubectl create -f -
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: event-display
+spec:
+  template:
+    spec:
+      containers:
+        - image: gcr.io/knative-releases/github.com/knative/eventing-sources/cmd/event_display
```

Comment:
should we point here to `gcr.io/knative-releases/knative.dev....` ? 

like:

```
gcr.io/knative-nightly/knative.dev/eventing-contrib/cmd/event_display@sha256:8734f54da846b8f95e71eb8422cdd1c10d451f76c77c16c8264d75ac25beee12
```

Reply:
thanks!